src/peergos/shared/user/CommittedWriterData.java

```diff
@@ -0,0 +1,16 @@
+package peergos.shared.user;
+
+import peergos.shared.merklebtree.*;
+
+public class CommittedWriterData {
```

Comment:
Except we have no way of creating the hash, because we don't know the chunking algorithm used by ipfs